matlab_archive/test/test_modeltime.m

```diff
@@ -0,0 +1,18 @@
+% test modeltime
+
+%% modeltime np2d_cond
+test_against_expected('np2d_cond', 36500000000, 18024418.611298);
+
+
+%% modeltime np3d_cond
+test_against_expected('np3d_cond', 5475000000, 1338452148.437500);
+
+
+function test_against_expected(fixture_name, expected_time, expected_deltime)
```

Comment:
Note this test function is much simpler than the others. The fact that the function doesn't write anything means we don't need to make a test directory.

Reply:
Not sure of meaning for expected_time (for routine to run?) and then expected_deltime (difference from earlier version)? Maybe you can tell me when we speak next. Thanks!

Reply:
Good question - no these refer to the times that I'm reading from the .hist file - so in Myrs etc. The test ensures that the function reading the file and inferring the final timestep (deltime) are what I expect from having inspected the files.

You can also write tests to check things like runtime as well, but that's less common that testing functionality like this.